lib/galaxy/dependencies/pipfiles/update.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+
+commit=0
+
+usage() {
+cat << EOF
+Usage: ${0##*/} [-c]
+
+Use pipenv to regenerate locked and hashed versions of Galaxy dependencies.
+Use -c to automatically commit these changes (be sure you have no staged git
+changes).
+
+EOF
+}
+
+while getopts ":i" opt; do
+    case "$opt" in
+        h)
+            usage
+            exit 0
+            ;;
+        i)
+            commit=1
+            ;;
+        '?')
+            usage >&2
+            exit 1
+            ;;
+    esac
+done
+
+THIS_DIRECTORY="$(cd "$(dirname "$0")" > /dev/null && pwd)"
+cd "$THIS_DIRECTORY"
+ENVS="develop
+flake8
+flake8_imports"
+
+for env in $ENVS
+do
+        cd "$env"
```

Comment:
Well, actually the other 2 are not needed only if you use `cd "$THIS_DIRECTORY/$env"` here.

Reply:
Right that was my comment on the other PR about not needing the absolute path anymore. Do you have a preference between these?

Reply:
Having both `&& pwd` and the `cd ..` is redundant, so I'd prefer to use `cd "$THIS_DIRECTORY/$env"` here and get rid of line 33 and 46.